### ISSUE TITLE
Let ISAR generate IsarTaskId

### ISSUE THREAD

**Issue by tsundvoll** [equinor/flotilla#1888]
Have the IsarTaskId for MissionTasks be generated by ISAR and reported back in the response when calling the start-mission endpoint in ISAR.

Similar to what is now done for IsarMissionId and IsarInspectionId.

This might require some refactoring, to have one object type be sendt to the ISAR insance (the mission request or specification), and one object instansiatied when the response comes back from ISAR.

**Comment by Eddasol** [comment]
Suggest accept that isar uses the provided id and only generate new it none is received. This is the current implementation and the issue should therefor be considered closed as not planned

**Comment by tsundvoll** [comment]
I agree to this approach, it simplifies it if Flotilla generates the IDs up front and ISAR just uses these. Then they can refer to the same things and not be confused.

**Comment by tsundvoll** [comment]
I agree to this approach, it simplifies it if Flotilla generates the IDs up front and ISAR just uses these. Then they can refer to the same things and not be confused.
